src/utils/settings.py

```diff
@@ -3,48 +3,29 @@
 import os
 
 
-def init(dataset: str):
+def init(data_config=None):
     """
+    Note:
+    - no typing of data_config possible (circular import)
+
```

Comment:
Didnt try it out, but isn't this working?

https://adamj.eu/tech/2021/05/13/python-type-hints-how-to-fix-circular-imports/